src/Core/Framework/DependencyInjection/app.xml

```diff
@@ -5,6 +5,7 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <parameters>
+        <parameter key="env(APP_URL)">http://localhost</parameter>
```

Comment:
Could you change it [here](https://github.com/shopware/platform/blob/96f67bfcc13b5b09e5ba360efaf105fae42d342a/src/Core/Framework/Resources/config/packages/shopware.yaml#L3) instead?

Reply:
@pweyck the bug it's fixed with that empty string as well. I created a new PR #1859 regarding just the change of the default value to `http://localhost`.